commit 785d8a3868fdd965299cd6e230803906b047eee0
Author: Clément Bœsch <ubitux@gmail.com>
Date:   Fri Feb 3 20:28:32 2012 +0100

    fate: use compression level 0 for flashsv2 tests.
    
    Output sizes mismatch (and thus MD5) between zlib version 1.2.5 and
    1.2.6 if compression level > 0. The issue doesn't affect the intra-only
    tests.

diff --git a/tests/codec-regression.sh b/tests/codec-regression.sh
index b8d27cb..87db24f 100755
--- a/tests/codec-regression.sh
+++ b/tests/codec-regression.sh
@@ -294,7 +294,7 @@ do_video_decoding "" "-pix_fmt yuv420p -sws_flags area+accurate_rnd+bitexact"
 fi
 
 if [ -n "$do_flashsv2" ] ; then
-do_video_encoding flashsv2.flv "-an -vcodec flashsv2 -sws_flags neighbor+full_chroma_int+accurate_rnd+bitexact -strict experimental"
+do_video_encoding flashsv2.flv "-an -vcodec flashsv2 -sws_flags neighbor+full_chroma_int+accurate_rnd+bitexact -strict experimental -compression_level 0"
 do_video_encoding flashsv2I.flv "-an -vcodec flashsv2 -sws_flags neighbor+full_chroma_int+accurate_rnd+bitexact -strict experimental -g 1"
 do_video_decoding "" "-pix_fmt yuv420p -sws_flags area+accurate_rnd+bitexact"
 fi
diff --git a/tests/ref/vsynth1/flashsv2 b/tests/ref/vsynth1/flashsv2
index 6c43bda..58e2e8f 100644
--- a/tests/ref/vsynth1/flashsv2
+++ b/tests/ref/vsynth1/flashsv2
@@ -1,5 +1,5 @@
-ec13d68cbce263f589fe79bcd2e98364 *./tests/data/vsynth1/flashsv2.flv
-8406137 ./tests/data/vsynth1/flashsv2.flv
+8d105780bb79915bc36f96f7a754f289 *./tests/data/vsynth1/flashsv2.flv
+9371095 ./tests/data/vsynth1/flashsv2.flv
 ed28c0021768629b2675e72bf1632426 *./tests/data/vsynth1/flashsv2I.flv
 8755723 ./tests/data/vsynth1/flashsv2I.flv
 efa88d09115a2e947eff00ee435ba3f3 *./tests/data/flashsv2.vsynth1.out.yuv
diff --git a/tests/ref/vsynth2/flashsv2 b/tests/ref/vsynth2/flashsv2
index b40988a..7102308 100644
--- a/tests/ref/vsynth2/flashsv2
+++ b/tests/ref/vsynth2/flashsv2
@@ -1,5 +1,5 @@
-6fbb1cf84d473a93035047c3a3e344ff *./tests/data/vsynth2/flashsv2.flv
-4814443 ./tests/data/vsynth2/flashsv2.flv
+3f4a7454fa664896ac7532e3551c4071 *./tests/data/vsynth2/flashsv2.flv
+9293862 ./tests/data/vsynth2/flashsv2.flv
 d5a22183d9fe670174340a8a1847e52a *./tests/data/vsynth2/flashsv2I.flv
 5037122 ./tests/data/vsynth2/flashsv2I.flv
 8f63e24049ba1789a7f8353c695a3d99 *./tests/data/flashsv2.vsynth2.out.yuv
